src/main/java/nl/tudelft/scrumbledore/keybinding/Keybinding.java

```diff
@@ -0,0 +1,95 @@
+package nl.tudelft.scrumbledore.keybinding;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javafx.scene.input.KeyCode;
+import nl.tudelft.scrumbledore.Constants;
+import nl.tudelft.scrumbledore.level.PlayerAction;
+
+/**
+ * Represents a Keybinding in the form of a key value pair.
+ * 
+ * @author Jeroen Meijer
+ */
+public class Keybinding {
+  private Map<KeyCode, PlayerAction> keyMap;
+  private Map<PlayerAction, KeyCode> actionMap;
```

Comment:
Are you sure it's not more efficient to make one hashmap?  This really does seem redundant, you can do both functions you mention with a single hashmap.

<review